app/javascript/sidebar-widget/SidebarWidget.jsx

```diff
@@ -40,8 +40,8 @@ class SidebarWidget extends Component {
       .then(json => {
         this.setState({ suggestedUsers: json });
       })
-      .catch(error => {
-        console.log(error);
+      .catch(() => {
+        this.setState({ suggestedUsers: [] });
```

Comment:
The `console.log(error)` we previously had wasn't great unless you happen to have the DevTools open in PROD when the error occurred. @mstruve added HoneyBadger to the project about a month ago I think, and this is what is used for server-side and client-side error reporting now. On the frontend, this has not been utilized yet aside from a global capture of all uncaught errors. The `catch` here is a great candidate to start using HoneyBadger.

In regards to the `setState` call, we're only masking the error and the user will have no idea why there are no suggested users. Instead of re-rendering an empty list of suggested users, a friendly error message would be better UX for the user. They'll see a rendered message like "Oops, it looks like we are having trouble loading suggested users right now."

That part though is out of the scope of this PR.

For the time being, I would suggest replacing the `console.log` with the HoneyBadger call.

```suggestion
        Honeybadger.notify(error);
```

Reply:
I added.